Source/Core/Core/PowerPC/PowerPC.h

```diff
@@ -57,7 +57,7 @@ struct tlb_entry
 };
 
 // This contains the entire state of the emulated PowerPC "Gekko" CPU.
-struct GC_ALIGNED64(PowerPCState)
+struct alignas(16) PowerPCState
```

Comment:
Important to note that this is changed to 16, as the only used instance of it is aligned to 16 bytes (and it's forbidden by the standard to use a smaller alignment for a type that has a stricter alignment).